setup.cfg

```diff
@@ -1,11 +1,11 @@
 [metadata]
 name = ara
 summary = ARA Records Ansible
-description-file =
+description_file =
     README.rst
 author = OpenStack Community
-author-email = openstack-discuss@lists.openstack.org
-home-page = https://github.com/ansible-community/ara
+author_email = openstack-discuss@lists.openstack.org
```

Comment:
unrelated but this makes me notice that we're still pointing to openstack-discuss even though ara has moved out of the openstack community's CI infrastructure a while back -- we can revisit this later